src/syscheckd/db/src/db.cpp

```diff
@@ -22,15 +22,15 @@ extern "C" {
  *
  * @return std::string Contains the dbsync's schema for FIM db.
  */
-const char * CreateStatement()
+static char *CreateStatement()
```

Comment:
This return should be std::string, I think that it is not a good idea to mix c and c ++ it seems to me

Reply:
We are using a C interface, so we should create C like function headers.
This change was requested by @Dwordcito in the review of this [PR](https://github.com/wazuh/wazuh/pull/10906#discussion_r756589952), but there was a bug that needed to be solved.